regulations/generator/title_parsing.py

```diff
@@ -42,8 +42,16 @@ def section(data):
     if len(data['index']) == 2 and data['index'][1][0].isdigit():
         element = {}
         element['is_section'] = True
-        element['label'] = '.'.join(data['index'])
         element['section_id'] = '-'.join(data['index'])
-        element['sub_label'] = re.search(
-            element['label'] + r'[^\w\[]*(.*)', data['title']).group(1)
+        if u"§§ " == data['title'][:3]:
+            element['is_section_span'] = True
+            match = re.match(
+                    r'([-.\w]*)' + r'[^\w\[]*(.*)', data['title'][3:])
```

Comment:
I know you didn't start this trend, but these regexes are rather cryptic. I think the intent of the original one was to strip any whitespace (including thin spaces) and the section marker. Can you think of a way to make these a bit more legible?